external/concealer/src/field_operator_application_for_autoware_universe.cpp

```diff
@@ -291,7 +291,13 @@ auto FieldOperatorApplicationFor<AutowareUniverse>::initialize(
         initial_pose_msg.header.stamp = get_clock()->now();
         initial_pose_msg.header.frame_id = "map";
         initial_pose_msg.pose.pose = initial_pose;
-        return setInitialPose(initial_pose_msg);
+
+        auto request =
+          std::make_shared<autoware_adapi_v1_msgs::srv::InitializeLocalization::Request>();
+        std::vector<geometry_msgs::msg::PoseWithCovarianceStamped> stamped_poses = {
+          initial_pose_msg};
+        request->pose.push_back(initial_pose_msg);
+        requestInitialPose(request);
```

Comment:
Did you forget to delete the unused variable `stamped_poses`? Or is it some kind of mistake?